### ISSUE TITLE
Feature Request: Erlang OTP 27 images

### ISSUE THREAD

**Issue by tomtaylor** [CircleCI-Public/cimg-elixir#158]
**Describe the Feature Request**

Add Erlang OTP 27 support to Docker images.

**Comment by kamidev** [comment]
What happened? There still doesn't seem to be working support for Erlang 27.

**Comment by andyleclair** [comment]
Bump, we also want this

**Comment by tpendragon** [comment]
Yes, very confused. Is there some support that needs to happen to get the bot moving here?

**Comment by marboledacci** [comment]
Hello everyone, we have finally released versions of the cimg/elixir with erlang 27
